cmd/infracost/report.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"github.com/infracost/infracost/internal/output"
+	"github.com/urfave/cli/v2"
+)
+
+func reportCmd() *cli.Command {
+	return &cli.Command{
+		Name:  "report",
+		Usage: "Combine and output multiple infracost JSON files",
+		UsageText: `infracost report [command options] [JSON paths...]
+
+Example:
```

Comment:
```suggestion
EXAMPLES:
```

That matches the other sections in --help, can we please change the other one to match this?